src/EA.Weee.Web/Areas/AatfReturn/Controllers/ReusedOffSiteController.cs

```diff
@@ -39,6 +39,13 @@ public virtual async Task<ActionResult> Index(Guid organisationId, Guid returnId
 
             await SetBreadcrumb(organisationId, BreadCrumbConstant.AatfReturn);
 
+            string path = (Request != null && Request.UrlReferrer != null) ? Request.UrlReferrer.AbsolutePath : String.Empty;
```

Comment:
I wouldn't worry doing this unless there is specific acceptance criteria... if they click back after selecting yes we should be ok with blank selection

Reply:
ok